src/gen_smtp_server_session.erl

```diff
@@ -342,10 +326,14 @@ handle_request({<<"HELO">>, <<>>}, State) ->
 handle_request({<<"HELO">>, Hostname},
 			   #state{options = Options, module = Module, callbackstate = OldCallbackState} = State) ->
 	case Module:handle_HELO(Hostname, OldCallbackState) of
+		{ok, 0, CallbackState} ->
```

Comment:
just opinion: might it be better to use some atom as switch for "unlimited"? We can use `infinity` for example (similar to Erlang's built-in `timeout()` datatype).

Reply:
I like the idea of using `infinity` - that also prevents accidental `0` configs.